firestore/google/cloud/firestore_v1/transaction.py

```diff
@@ -200,6 +202,37 @@ def _commit(self):
         self._clean_up()
         return list(commit_response.write_results)
 
+    def get_all(self, references):
+        """Retrieves multiple documents from Firestore.
+
+        Args:
+            references (List[.DocumentReference, ...]): Iterable of document
+                references to be retrieved.
+
+        Yields:
+            .DocumentSnapshot: The next document snapshot that fulfills the
+            query, or :data:`None` if the document does not exist.
+        """
+        return self._client.get_all(references, transaction=self._id)
+
+    def get(self, ref_or_query):
+        """
+        Retrieve a document or a query result from the database.
+        Args:
+            ref_or_query The document references or query object to return.
```

Comment:
ref_or_query 
                (:class:`~google.cloud.firestore_v1.document.DocumentReference` or 
                :class:`~google.cloud.firestore_v1.query.Query`):
                The document references or query object to return.

Reply:
actually in the project prepared a  doc with two patterns one is declare `type and :param` and other what i use here that you can  in the whole file , so to match that pattern i use this format, if i change that than may be need to change for whole file.

Reply:
Ok, go ahead with what you have added.